### PR TITLE
Pipeline, Anna Azimkova - 22FPL1

### PR DESCRIPTION
**Pull request by puglyann**

### CODE REVIEW

**Comment by marina-kaz** [comment]
No pipeline changes for me to review today

File: seminars/seminar_05_13_2024/analyzed_text.conllu
Comment (marina-kaz) [inline seminars/seminar_05_13_2024/analyzed_text.conllu:1]: Please remove changes

File: lab_6_pipeline/pipeline.py [hunk 1]
```diff
@@ -27,16 +37,48 @@ def __init__(self, path_to_raw_txt_data: pathlib.Path) -> None:
...
+        if len(raw_files) != len(meta_files):
+            raise InconsistentDatasetError
+
+        sorted_raw_files = sorted(raw_files, key=get_article_id_from_filepath)
+        sorted_meta_files = sorted(meta_files, key=get_article_id_from_filepath)
+
+        for i, (raw, meta) in enumerate(zip(sorted_raw_files, sorted_meta_files)):
+            if i + 1 != get_article_id_from_filepath(raw) or \
+                    i + 1 != get_article_id_from_filepath(meta) or \
+                    any(file.stat().st_size == 0 for file in (raw_files + meta_files)):
```
Comment (marina-kaz) [inline lab_6_pipeline/pipeline.py:70]: ` == 0` is redundant

**Review by marina-kaz — COMMENTED** [review]
Good start!
Try to get at least mark 8.

File: seminars/seminar_05_13_2024/analyzed_text.conllu
Comment (puglyann) [reply seminars/seminar_05_13_2024/analyzed_text.conllu:1]: done

**Pull request closed without merging**